packages/backend/src/document/document-image-refs.ts

```
@@ -0,0 +1,238 @@
+  const rehostable = new Map<string, ImageRef>();
+  const foreign = new Map<string, ImageRef>();
+  let truncated = false;
+
+  const visit = (node: unknown, depth: number): void => {
+    if (depth > MAX_DEPTH) {
+      truncated = true;
+      return;
+    }
+    const ref = parseImageRef(node, trustedOrigin);
+    if (ref) {
+      const bucket = isRehostable(ref, sourceWorkspaceId) ? rehostable : foreign;
```

ESLint

[error] Replace `·?·rehostable` with `⏎········?·rehostable⏎·······`

(prettier/prettier)